config/formats.ts

```diff
@@ -4478,6 +4478,37 @@ export const Formats: import('../sim/dex-formats').FormatList = [
 		section: "Past Generations",
 		column: 4,
 	},
+	{
+		name: "[Gen 3] Orre Colosseum",
+		threads: [
+			`&bullet; <a href="https://www.smogon.com/forums/threads/3666831/">ADV Doubles OU</a>`,
+			`&bullet; <a href="https://www.smogon.com/forums/threads/orre-colosseum-now-playable.3698894/"> Orre Colosseum</a>`,
+			`&bullet; <a href-"https://discord.gg/cSZE6MZX5Q">Join the discord server</a>`,
+		],
+
+		mod: 'gen3colosseum',
+		gameType: 'doubles',
+		ruleset: ['Obtainable', 'Team Preview', 'Species Clause', 'Stadium Sleep Clause', 'Freeze Clause Mod', 'Max Team Size = 6', 'VGC Timer',
+			'Nickname Clause', 'Endless Battle Clause', 'Cancel Mod', 'Picked Team Size = 4', 'Exact HP Mod', "Item Clause = 1", 'Open Team Sheets'],
+		banlist: ['Soul Dew', 'Deoxys-Defense', 'Deoxys-Attack', 'Deoxys-Speed'],
+		unbanlist: ['Latios', 'Latias', 'Wobbuffet', 'Wynaut'],
+		restricted: ['Restricted Legendary', "Mythical"],
+		bestOfDefault: true,
+		rated: true,
+		onBegin() {
+			this.add('rule', 'Self-KO clause: If your last pokemon faints to a self-KO move or effect you will lose the battle');
+		},
+		onValidateTeam(team) {
+			const restrictedSpecies = [];
+			for (const set of team) {
+				const species = this.dex.species.get(set.species);
+				if (this.ruleTable.isRestrictedSpecies(species)) restrictedSpecies.push(species.name);
+			}
+			if (restrictedSpecies.length > 0) {
+				return [`You can't use any restricted Pok\u00E9mon (you have: ${restrictedSpecies.join(', ')})`];
```

Comment:
Wouldn't it be better to just ban these pokemon?

Reply:
Good catch. They had restricted mon in it at one point but that got split into a new format (One that may be added later but that is a pull request for another day)